.github/workflows/validate.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: cmake --build build --verbose --target all -j 2
 
       - name: Install
-        run: cmake --install build
+        run: pip3 install -v build/.
```

Comment:
I would also like to test both install mechanism somehow.

Reply:
I do not like duplication. One has to maintain two things and they can diverge. I prefer to stick to one way of installing espressopp, either via `make install` or `pip`.

Reply:
Spack and the rpm packages on openSUSE and fedora (https://github.com/espressopp/fedora-copr) use cmake to install, so I would really like to keep that.

Reply:
I am not familiar in building spack or rpm packages. If they cannot use `pip` then let's stick to our hand-crafted `make install`.

Reply:
Everything is possible in spack and rpm, using standard cmake just makes it much easier as spack has a cmake class and rpm has cmake macros.

Plus our `make install` is pretty standard already. E.g. it supports `DESTDIR`.

Reply:
Ok.

Reply:
Do we want to support both installation methods? Then we have to explain what the difference between `make install` and `pip install` is and keep it up-to-date.